components/movementsensor/gpsrtkpmtk/gpsrtkpmtk.go

```diff
@@ -354,6 +355,15 @@ func (g *rtkI2C) receiveAndWriteI2C(ctx context.Context) {
 		g.logger.Infof("caster %s seems to be down", g.ntripClient.URL)
 	}
 
+	srctable, err := g.ntripClient.Client.ParseSourcetable()
+	if err != nil {
+		g.logger.Errorf("failed to get source table: %v", err)
+	}
+	nmea, nmeaerr := findLineWithMountPoint(srctable, g.ntripClient.MountPoint)
+	if nmeaerr != nil {
+		g.logger.Errorf("can't find mountpoint in source table, found err %v\n", nmeaerr)
```

Comment:
Same question here, but also I wouldn't put a newline in the error text, because that will make the logging look bad.